app/services/csv_exporter.rb

```diff
@@ -47,6 +48,7 @@ def to_csv #rubocop:disable Metrics/CyclomaticComplexity, Metrics/MethodLength
           @kase.internal_deadline&.strftime('%F'),
           @kase.external_deadline&.strftime('%F'),
           @kase.date_responded.present? ? @kase.date_responded.strftime('%F') : nil,
+          @kase.date_draft_compliant.present? ? @kase.date_draft_compliant.strftime('%F') : nil,
```

Comment:
might be out of scope, but wondering if we shouldn't be using a named to_s overload on dates rather than calling strftime('%F') everywhere?